src/xdocs/writingchecks.xml

```diff
@@ -757,30 +756,30 @@ public class LimitImplementationFiles extends AbstractFileSetCheck
 
       <p>
         Note that the configuration via bean introspection also applies
-        here.  By implementing the <code>setMax()</code>
+        here. By implementing the <code>setMax()</code>
         method the FileSetCheck automatically makes &quot;max&quot; a
         legal configuration parameter that you can use in the Checkstyle
         configuration file.
       </p>
       <p>
-        There are virtually no limits what you can do in
+        There are virtually no limits to what you can do in
         FileSetChecks, but please do not be crazy.
       </p>
 
     </section>
 
-    <section name="Declare check's external resource locations">
+    <section name="Declare checks external resource locations">
```

Comment:
Originally was fine to me. We are saying the possessive of the check. The resource locations of the check.

Reply:
Please revert this change